### PR TITLE
removing better-sqlite3 library and types

### PR DESCRIPTION
**Pull request by christianllv**
**Please check if the PR fulfills these requirements**
- [x] Followed the [Contributing](https://github.com/jaredwray/keyv/blob/main/CONTRIBUTING.md) guidelines.
- [x] Tests for the changes have been added (for bug fixes/features) with 100% code coverage.
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Fix issue #669

### CODE REVIEW

**Comment by christianllv** [comment]
@jaredwray please check, this change fix the issue #669

**Comment by jaredwray** [comment]
@christianllv - this is not the right fix as we do not use better-sqlite anymore. Can you please update the index.d.ts file to remove that and validate that it is working.

File: packages/sqlite/package.json [hunk 1]
```diff
@@ -56,6 +56,8 @@
 	},
 	"homepage": "https://github.com/jaredwray/keyv",
 	"dependencies": {
+		"@types/better-sqlite3": "^7.6.3",
```
Comment (jaredwray) [inline packages/sqlite/package.json]: These lines should be removed as we do not have a dependency on this

Comment (chrisllontop) [reply packages/sqlite/package.json]: Fixed

**Pull request merged**